schemas/purge-cache-request-list.yml

```diff
@@ -0,0 +1,48 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "Open Purge Request List"
+description: |
+  A list of currently open purge-cache requests.
+type:         object
+properties:
+  continuationToken:
+    type: string
+    description: Passed back from Azure to allow us to page through long result sets.
+  cacheHit:
+    type: boolean
+    description: True if the cache has been used in this request.
+  requests:
+    type:         array
+    description: |
+      A simple list of purge-cache requests.
+    items:
+      type: object
+      properties:
+        provisionerId:
+          type: string
+          description: ProvisionerId associated with the workerType.
+        workerType:
+          type: string
+          description: Workertype cache exists on.
+        cacheName:
+          type: string
+          description: Name of cache to purge.
+        expires:
```

Comment:
as far as the worker is concerned this doesn't expire.. This is an internal thing because we don't really want to keep the log around forever... but as far as the worker is aware this is an append only log...